sample/put_object.go

```diff
@@ -66,32 +66,32 @@ func PutObjectSample() {
 	}
 	fmt.Println("Object Meta:", props)
 
-	// 场景6：大文件分片上传，支持并发上传，断点续传功能。
-	// 分片上传，分片大小为100K。默认使用不使用并发上传，不使用断点续传。
+	// case 6：big file's multipart upload. It supports concurrent upload with checkpoint
+	// multipart upload with 100K as part size. By default 1 thread is used and no checkpoint is used.
 	err = bucket.UploadFile(objectKey, localFile, 100*1024)
 	if err != nil {
 		HandleError(err)
 	}
 
-	// 分片大小为100K，3个协程并发上传。
+	// part size is 100K and three threads are used
 	err = bucket.UploadFile(objectKey, localFile, 100*1024, oss.Routines(3))
 	if err != nil {
 		HandleError(err)
 	}
 
-	// 分片大小为100K，3个协程并发下载，使用断点续传上传文件。
+	// part size is 100K and three threads are used with checkpoint
 	err = bucket.UploadFile(objectKey, localFile, 100*1024, oss.Routines(3), oss.Checkpoint(true, ""))
 	if err != nil {
 		HandleError(err)
 	}
 
-	// 断点续传功能需要使用本地文件，记录哪些分片已经上传。该文件路径可以Checkpoint的第二个参数指定，如果为空，则为上传文件目录。
+	// specifies the local file path for checkpoint files.
```

Comment:
// specifies the local file path for checkpoint files. 
	// the 2nd paramter of Checkpoint can specify the file path, when the file path is empty, it will upload the directory.